src/pages/about-us.js

```diff
@@ -1,899 +1,349 @@
 /* eslint no-undef: 0 */
 import React from 'react'
+import { graphql } from 'gatsby'
 
 import Layout from '../components/layout'
 import styled from 'styled-components'
 import Link from '../components/navigation/Link'
 import { LinkButton } from '../components/buttons'
 import Section, { TopSection } from '../components/layout/Section'
 import Grid, { Col, Row } from '../components/layout/Grid'
-import { H2, H2Ref, H3, P } from '../components/text'
+import { H2, H2Ref, H3, P, H5 } from '../components/text'
 import { UpcomingTrainingSection } from '../components/training'
 import Ul, { Li } from '../components/layout/Ul'
 import Header from '../components/layout/Header'
 import CallToActionNextTrainings from '../components/layout/CallToActionNextTrainings'
-import { Card, Video, Image, Newsletter } from '../components/elements'
-import { Blockquote } from '../components/text'
-import {
-  RICHARD,
-  HORACIO,
-  WILL,
-  EVA,
-  ROY,
-  DAVID,
-  FRANCISCO,
-  PAUL,
-  ABOUT_VALUES,
-} from '../config/images'
+import { Card, Video, Image } from '../components/elements'
 import { HideComponentsUsingCss } from '../components/utils'
 import trackUserBehaviour, {
   SOCIAL_NETWORK_LINK_CLICK,
 } from '../components/utils/trackUserBehaviour'
 import { SCREEN_SM_MAX } from '../components/utils'
 import LeanJSsprints from '../components/elements/LeanJSsprints'
+import Box from '../components/layout/Box'
+import { GREY } from '../config/styles.js'
 
-const CoachTitle = styled(H3)`
-  & {
-    padding-top: 0;
+const CoachProfiles = styled(Box)`
+  h5 {
+    margin-bottom: 0.5rem;
```

Comment:
@ric9176 can you please use the design system for the spacing? you don't have to create a new component, you can  <H5 mt={1} />  being 1 because of index 1 in the spacing https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/master/src/config/styles.js#L61